templates/tfengine/recipes/project.hcl

```diff
@@ -51,20 +56,22 @@ schema = {
       ]
       properties = {
         project_id = {
-          description = "ID of project to create and/or provision resources in."
-          type        = "string"
-          pattern     = "^[a-z][a-z0-9\\-]{4,28}[a-z0-9]$"
+          description       = "ID of project to create and/or provision resources in."
```

Comment:
why additional spaces before the `=`

Reply:
My bad, I leave the extra spaces because I added a `terraformPattern` that was removed.

Fixed